test/page_objects/blog_posts/index_page.rb

```diff
@@ -7,6 +7,15 @@ class IndexPage < AePageObjects::Document
     collection :blog_posts, locator: 'tbody', item_locator: 'tr' do
       element :title, locator: '.js-blog-title'
       element :body, locator: '.js-blog-body'
+      element :delete_button, locator: '.js-delete'
+
+      def delete!
+        delete_button.node.click
+        alert = node.base.driver.browser.switch_to.alert
+        yield alert if block_given?
+        alert.accept
+        window.change_to(IndexPage)
```

Comment:
For an intro class this is perfect, but discussing how to handle this if you wanted them to cancel instead of accept might make for an interesting conversation